CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1 @@
-## Code of Conduct
-This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct).
-For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq) or contact
-opensource-codeofconduct@amazon.com with any additional questions or comments.
+[Code of Conduct](docs/content/en/docs/community/code_of_conduct.md)
```

Comment:
Should we have these point to the website instead of GitHub?

Reply:
Personally I prefer them to exist in Github. The docs can provide further content but if I have the code checked out locally it's easier to read plain markdown in my IDE

Reply:
Yup that's why I linked them this way